skimage/transform/_geometric.py

```diff
@@ -226,15 +226,21 @@ def estimate(self, src, dst):
 
         Number of source and destination coordinates must match.
 
-        The transformation is defined as::
+        The transformation is defined as:
 
-            X = (a0*x + a1*y + a2) / (c0*x + c1*y + 1)
-            Y = (b0*x + b1*y + b2) / (c0*x + c1*y + 1)
+        .. math::
 
-        These equations can be transformed to the following form::
+            X = (a_0\,x + a_1\,y + a_2)/(c_0\,x + c_1\,y + 1)
```

Comment:
It should work, [apparently](http://matplotlib.org/sampledoc/extensions.html).
Have you tried this https://stackoverflow.com/questions/16468397/mathjax-expression-in-sphinx-python-not-rendering-correclty ?